src/test/java/codeu/model/store/basic/MessageStoreTest.java

```diff
@@ -84,8 +85,8 @@ public void testAddMessage() {
   public void testGetMessageCount(){
     int messageCount = messageStore.getMessageCount();
 
-    // test before message added, expected result = 0
-    Assert.assertEquals(0, messageCount);
+    // test before message added, expected result = 3 (3 added from setup)
+    Assert.assertEquals(3, messageCount);
```

Comment:
I am a bit confused why messageCount = 3 here. I understand that in testGetConversationCount, a new conversation is created and that's why count changes from 1 to 2. I am not sure where new messages are added before this line though.

Reply:
In the function setup()
- new array is created 
- 3 msg were added there 
- the array list is passed to the setMessage() function 
so when i get the size, it kinda came with the 3 msg already added. That's why i added another msg to it to see if it works the way i want it too

Reply:
Oh I see, that makes sense, thank you!